### PR TITLE
Update golden files for operator

### PR DESCRIPTION
**Pull request by howardjohn**
Over time these fall out of sync which makes it harder to make and test
changes to the logic

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Comment by howardjohn** [comment]
@ostromart

File: operator/cmd/mesh/testdata/manifest-generate/data-snapshot/charts/base/files/gen-istio-cluster.yaml [hunk 1]
```diff
@@ -3276,3 +3277,368 @@ spec:
     storage: true
 
 ---
+
+---
+# Source: crds/crd-mixer.yaml
```
Comment (richardwxn) [inline operator/cmd/mesh/testdata/manifest-generate/data-snapshot/charts/base/files/gen-istio-cluster.yaml:3282]: why we still have the mixer related CRDs here

Comment (howardjohn) [reply operator/cmd/mesh/testdata/manifest-generate/data-snapshot/charts/base/files/gen-istio-cluster.yaml:3282]: Its just copying from manifests/. They were removed in a later PR, I will update this one again.

**Comment by howardjohn** [comment]
@richardwxn ptal

**Review by richardwxn — APPROVED** [review]

**Review by stevenctl — APPROVED** [review]

**Pull request merged**